node_modules/gh-orgunit/lib/internal/dao.js

```diff
@@ -760,3 +763,402 @@ var _exportOrgUnitJSON = function(rootId, orgUnits, callback) {
+
+        // Import the organisational unit's child organisational units (if any)
+        _importOrgUnitChildren(orgUnit.children, createdOrgUnit, function(err) {
+            if (err) {
+                return callback(err);
+            }
+
+            // Import the series under this organisational unit
+            _importOrgUnitSeries(orgUnit.series, createdOrgUnit, function(err) {
+                if (err) {
+                    return callback(err);
+                }
+
+                // TODO: Import events directly associated to organisational units
+                return callback();
+            });
+        });
+    });
+};
+
+/**
+ * Update or create an organisational unit
+ *
+ * @param  {Object}     orgUnitToImport             The organisational unit to import
+ * @param  {Object}     parent                      The paren organisational unit to which the organisational unit belongs
+ * @param  {Function}   callback                    Standard callback function
+ * @param  {Object}     callback.err                An error object, if any
+ * @param  {Event}      callback.persistedOrgUnit   The persisted organisational unit
+ * @api private
+ */
+var _updateOrCreateOrgUnit = function(orgUnitToImport, parent, callback) {
+    if (orgUnitToImport.id) {
+        // Update the organisational unit
+        getOrgUnit(orgUnitToImport.id, false, function(err, persistedOrgUnit) {
+            if (err) {
+                return callback(err);
+            }
+
+            var update = {
+                'displayName': orgUnitToImport.displayName,
+                'description': orgUnitToImport.description,
+                'type': orgUnitToImport.type,
+                'metadata': orgUnitToImport.metadata,
+                'published': orgUnitToImport.published
+            };
+            return updateOrgUnit(persistedOrgUnit, update, callback);
+        });
+    } else {
+        // Create the organisational unit
+        var opts = {
+            'description': orgUnitToImport.description,
+            'externalId': orgUnitToImport.externalId,
+            'metadata': orgUnitToImport.metadata,
+            'published': orgUnitToImport.published,
+            'parentId': parent.id
+        };
+        return createOrgUnit(orgUnitToImport.AppId, orgUnitToImport.displayName, orgUnitToImport.type, parent.GroupId, opts, callback);
+    }
+};
+
+/**
+ * Import the child organisational units of an organisational unit
+ *
+ * @param  {Object[]}   children        The child organisational units to import
+ * @param  {OrgUnit}    parent          The organisational unit under which the child organisational units should be imported
+ * @param  {Function}   callback        Standard callback function
+ * @param  {Object}     callback.err    An error object, if any
+ * @api private
+ */
+var _importOrgUnitChildren = function(children, parent, callback) {
+    children = children || [];
+    if (_.isEmpty(children)) {
+        return callback();
+    }
```

Comment:
If we're checking `_.isEmpty(children);`, then `children = children || [];` isn't necessary.